docs/payid-discovery.md

```diff
@@ -0,0 +1,46 @@
+---
+id: payid-discovery
+title: Discover a PayID
+sidebar_label: Discover a PayID
+---
+
+The PayID Discovery endpoint facilitates finding the PayID URI for a given PayID.
+
+## PayID discovery by a wallet application
+
+Suppose Alice wants to send Bob some XRP from a web-based wallet provider that Alice has an account on. Alice logs in to the wallet provider and enter Bob's PayID, such as `bob$receiver.example.com`, into the wallet UI to start the payment. If the wallet application is set up to use the PayID Discovery endpoint, the wallet application performs a WebFinger query that looks for the PayID Discovery service provider, like this:
+
+```
+GET /.well-known/webfinger?resource=payid%3Abob%24receiver.example.com
+HTTP/1.1
+Host: receiver.example.com
+```
+
+If the server where Bob receives currency supports the PayID Discovery endpoint, it responds like this:
```

Comment:
To avoid this "if", maybe revise the opening paragraph to set up the situation a little more, e.g. "Alice has an account at Example Wallet and Bob as an account at Foo Exchange. Both support the PayID Discovery endpoint..."